src/backend/core/models.py

```diff
@@ -205,7 +221,7 @@ class User(AbstractBaseUser, BaseModel, auth_models.PermissionsMixin):
 
     objects = auth_models.UserManager()
 
-    USERNAME_FIELD = "admin_email"
+    USERNAME_FIELD = "sub"
```

Comment:
Results in 
![image](https://github.com/numerique-gouv/people/assets/30848497/c5012790-cbfe-449e-b130-f834e2790086)

While it's accurate, I'd like to customize it to "Sub or username" to make it a bit clearer.

Reply:
as discussed IRL we keep it like this as it is very "insider"